.github/workflows/build-common.yml

```diff
@@ -121,16 +118,6 @@ jobs:
           name: integration-test-results
           path: jmx-metrics/build/reports/tests/integrationTest
 
-  link-check:
```

Comment:
move into the callers so they can choose not to block merging / snapshots if it fails